catalog/rest/catalog-rest-service-impl/src/main/java/org/codice/ddf/rest/service/impl/CatalogServiceImpl.java

```diff
@@ -973,10 +967,10 @@ private void parseAttribute(
       Map<String, AttributeImpl> attributeMap,
       String parsedName,
       InputStream inputStream,
-      AttributeType.AttributeFormat attributeFormat) {
+      AttributeFormat attributeFormat) {
     try (InputStream is = inputStream;
         InputStream boundedStream = new BoundedInputStream(is, MAX_INPUT_SIZE + 1L)) {
-      if (attributeFormat == OBJECT) {
+      if (attributeFormat == AttributeFormat.OBJECT) {
```

Comment:
:question: do these need to be the raw enum type? i remember seeing an error that said it can't be qualified, unsure tho. i guess if it compiles, it's okay